nuxeo-services/login/nuxeo-platform-login-saml2/src/main/java/org/nuxeo/ecm/platform/auth/saml/SAMLConfiguration.java

```diff
@@ -105,6 +109,11 @@ public static boolean getWantAssertionsSigned() {
         return Boolean.parseBoolean(Framework.getProperty(WANT_ASSERTIONS_SIGNED));
     }
 
+    public static int getSkewTimeMillis() {
+        String skewTimeMs = Framework.getProperty(SKEW_TIME_MS);
```

Comment:
At some point we must think about refactoring the SAML config to use a proper service + extension point instead of ad hoc system properties. Even using the configuration service would be much better.